core/pom.xml

```diff
@@ -60,6 +60,33 @@
       </build>
     </profile>
   </profiles>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
+        <version>3.5.0</version>
+
+		<executions>
```

Comment:
This should use 2-space indent consistently.

Reply:
I replaced tabs by spaces